### PR TITLE
update RBAC for OnCall documentation

### PR DESCRIPTION
**Pull request by joeyorlando**
This PR simply removes the RBAC for OnCall table of roles/granted actions + adds a link to this information which already exists in the Grafana OnCall documentation.

I would suggest we keep that information in the OnCall docs because it makes it easier to update in the event of an RBAC related code change (+ we have [a script](https://github.com/grafana/oncall/blob/dev/docs/sources/user-and-team-management/convert_plugin_json_rbac_roles_into_markdown_table.py) which reads our plugin's `plugin.json` and regenerates this table in the docs 🤓 )

### CODE REVIEW

**Review by IevaVasiljeva — APPROVED** [review]
Oh nice, I didn't realise you had a list of roles and actions in your docs.

File: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md [hunk 1]
```diff
@@ -127,38 +127,8 @@ This feature is behind the `accessControlOnCall` feature toggle.
...
-| `plugins:grafana-oncall-app:settings-editor`              | `plugins.app:access`<br>`grafana-oncall-app.other-settings:read`<br>`grafana-oncall-app.other-settings:write`                                                                                                                                                                                                                                                                                                                                                         | Read and update OnCall settings.                                                                                                  |
+If you are using [Grafana OnCall](https://grafana.com/docs/oncall/latest/get-started/), you can try out the integration between Grafana OnCall and RBAC. For a detailed list of the available OnCall RBAC roles, see the table in the Grafana
+OnCall documentation [here](https://grafana.com/docs/oncall/latest/user-and-team-management/#available-grafana-oncall-rbac-roles--granted-actions).
```
Comment (jdbaldry) [inline docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:131]: - Semantic line breaks for better line based diffing in the GitHub UI in the future. (https://sembr.org)
- Prefer "refer to" over "see" (https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/ReferTo.yml#L3)
- Avoid generic reference "here" (Available Grafana OnCall RBAC roles + granted actions)

```suggestion
If you are using [Grafana OnCall](https://grafana.com/docs/oncall/latest/get-started/), you can try out the integration between Grafana OnCall and RBAC.
For a detailed list of the available OnCall RBAC roles, refer the table in [Available Grafana OnCall RBAC roles and granted actions](https://grafana.com/docs/oncall/latest/user-and-team-management/#available-grafana-oncall-rbac-roles--granted-actions).
```

Comment (joeyorlando) [reply docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md]: thanks for the tips @jdbaldry! 🙏

Comment (jdbaldry) [reply docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md]: No problemo! I'll get the Vale linter to do this automatically for us soon :)

Thanks for incorporating the feedback!

**Comment by joeyorlando** [comment]
on a side note, how can I get an approved `grafana/grafana` merged? 👀

**Comment by IevaVasiljeva** [comment]
I just merged it for you, do you not have the permissions to do it yourself?

**Comment by joeyorlando** [comment]
thank you @IevaVasiljeva 🙂 

> do you not have the permissions to do it yourself?

I don't believe so. It mentioned it could only be merged by certain users.

**Comment by IevaVasiljeva** [comment]
> I don't believe so. It mentioned it could only be merged by certain users.

Something to dig into for your next Grafana PR :)

**Pull request merged**